app/model/Event/EducationTerm.php

```diff
@@ -37,4 +40,21 @@ public function getEndDate(): Date
     {
         return $this->endDate;
     }
+
+    /** @param array<EducationTerm> $terms */
+    public static function countTotalDays(array $terms): int
```

Comment:
A tohle nekde pise skautis ne? tak bych to nerad pocital a jen si to bral z něj

Reply:
To je jen přesunuté tohle: https://github.com/skaut/Skautske-hospodareni/pull/2303#discussion_r1346209539

Reply:
Jinými slovy ne :D

Reply:
ze skautis tedy nikde tohle cislo neuvadi? tak potom jo no

Reply:
Neuvádí :(